arapp.json

```diff
@@ -13,5 +11,11 @@
       "params": []
     }
   ],
-  "path": "contracts/CounterApp.sol"
+  "path": "contracts/CounterApp.sol",
+  "environments": {
+    "default": {
+      "network": "development",
+      "appName": "app.aragonpm.eth"
```

Comment:
We should make sure aragonCLI probably changes the name of this on `aragon init` (not sure if this logic was added in as part of the change to make `environments` happen.

Reply:
Yes, this was added as part of the changes